base/filesystem.jl

```diff
@@ -4,6 +4,37 @@
 
 module Filesystem
 
+"""
+    JL_O_APPEND
+    JL_O_ASYNC
+    JL_O_CLOEXEC
+    JL_O_CREAT
+    JL_O_DIRECT
+    JL_O_DIRECTORY
+    JL_O_DSYNC
+    JL_O_EXCL
+    JL_O_LARGEFILE
+    JL_O_NOATIME
+    JL_O_NOCTTY
+    JL_O_NONBLOCK
+    JL_O_PATH
+    JL_O_RANDOM
+    JL_O_RDONLY
+    JL_O_RDWR
+    JL_O_RSYNC
+    JL_O_SEQUENTIAL
+    JL_O_SHORT_LIVED
+    JL_O_SYNC
+    JL_O_TEMPORARY
+    JL_O_TMPFILE
+    JL_O_TRUNC
+    JL_O_WRONLY
+
+Enum constant for the `open` syscall, where `JL_O_*` corresponds to the `O_*` constant.
```

Comment:
This is not an "enum" in the sense of a Julia `@enum` value.  They are just `UInt32` constants.

(For some reason, `JL_O_TMPFILE` is a `UInt16`, though??)

Reply:
yeah, a lot of these are defined incorrectly now, including being the wrong type in some cases, and that they don't follow the UV names that are now consistently defined for them instead our current ad-hoc definitions